### PR TITLE
Feat/can 91/new crowd action designs

### PR DESCRIPTION
**Pull request by wizlif**
1. Spotlight header design implementation.
2. Share collAction.
3. CrowdAction details.
4. Modal design
5. Flutter version upgrade

### CODE REVIEW

File: lib/presentation/crowd_action/pages/crowd_action_details.dart [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+                          ?.copyWith(fontWeight: FontWeight.bold),
+                    ),
+                    const SizedBox(
+                      height: 20,
+                    ),
+                    Wrap(
+                      spacing: 12.0,
+                      children: [
+                        SignUpChip(),
+                        if (crowdAction.tags?.isNotEmpty == true)
```
Comment (rubenhorn) [inline lib/presentation/crowd_action/pages/crowd_action_details.dart:94]: Maybe will have a category (and sub-category instead).
👉 Check if the design would change in that case.

File: lib/presentation/crowd_action/pages/widgets/share_collaction_card.dart [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            const SizedBox(
+              height: 10,
+            ),
+            PillButton.icon(
+              text: "Share CollAction",
+              leading: const Icon(
+                Icons.person_outline,
+              ),
+              onTap: () => Share.share(
+                'Share app title',
```
Comment (rubenhorn) [inline lib/presentation/crowd_action/pages/widgets/share_collaction_card.dart:72]: Mark with TODO

File: lib/presentation/crowd_action/pages/crowd_action_details.dart (hunk 1 above)
Comment (wizlif) [reply lib/presentation/crowd_action/pages/crowd_action_details.dart:94]: Nope, it wouldn't change

File: lib/domain/crowdaction/crowdaction.dart [hunk 3]
```diff
@@ -13,10 +13,11 @@ abstract class CrowdAction implements _$CrowdAction {
...
     // required String subtitle,
     // required String description,
-    // required int numParticipants,
+    String? image, // TODO - Request backend to add this
+    required int numParticipants,
     // required int participantsGoal,
     // DateTime? startDate,
     // DateTime? endDate,
-    // List<String>? tags,
+    List<String>? tags, // TODO - Request backend to add this
```
[the remark this answers is not in the document]
Comment (wizlif) [reply lib/domain/crowdaction/crowdaction.dart:21]: I've added category & sub category to crowd action dto.

**Review by rubenhorn — APPROVED** [review]

**Pull request merged**